api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,5 +117,22 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+	<changeSet id="TRUNK-4791-20180214-1" author="patrick">
+		<preConditions onFail="MARK_RAN">
```

Comment:
You need to create a separate changeset for each column.

Reply:
Fixed